quickcheck-instances.cabal

```diff
@@ -103,7 +105,7 @@ library
     , transformers-compat   >=0.6.5   && <0.7
     , unordered-containers  >=0.2.2.0 && <0.3
     , uuid-types            >=1.0.3   && <1.1
-    , vector                >=0.9     && <0.13
+    , vector                >=0.9     && <0.14
```

Comment:
vector-0.13 is not on Hackage, There is no guarantee what the actual `vector-0.13` will contain, so this relaxation is not ok.